src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -235,6 +235,23 @@ public function getEnabledTransitions(object $subject)
         return $enabledTransitions;
     }
 
+    public function getEnabledTransition(object $subject, string $name): ?Transition
+    {
+        $marking = $this->getMarking($subject);
+
+        foreach ($this->definition->getTransitions() as $transition) {
+            if ($transition->getName() !== $name) {
+                continue;
+            }
+            $transitionBlockerList = $this->buildTransitionBlockerListForTransition($subject, $marking, $transition);
+            if ($transitionBlockerList->isEmpty()) {
+                return $transition;
```

Comment:
What if many transition match?
cf https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Workflow/Workflow.php#L199

Reply:
:eyes: it's possible to have many transitions with the same name ? I did not realize...
In this case, I would have to add 2 optional arrays, one for the `froms` and one from the `tos`.
This way, it would be possible to identify specifically any transition.

I can try to do something, but I think it's overkill.

Reply:
So now, it is possible to retrieve any specific transition.
What do you think ?